pyproject.toml

```diff
@@ -0,0 +1,78 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "tuspy"
+description = "A Python client for the tus resumable upload protocol ->  http://tus.io"
+authors = [
+    {name = "Ifedapo Olarewaju", email="ifedapoolarewaju@gmail.com"},
+]
+
+# Dependency Information
+requires-python = ">=3.8"
+dependencies = [
+    "requests>=2.18.4",
+    "six>=1.11.0",
+    "tinydb>=3.5.0",
+    "aiohttp>=3.6.2",
+]
+
+# Extra Information
+readme = "README.md"
+license = {text = "MIT Licence"}
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Web Environment",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Communications :: File Sharing",
+    "Topic :: Internet :: File Transfer Protocol (FTP)",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Typing :: Typed",
+]
+
+# Misc
+dynamic = ["version"]
+
+[project.urls]
+Homepage = "http://tus-py-client.readthedocs.io/en/latest/"
```

Comment:
It doesn't look like the [documentation online has been updated in over 5 years](https://readthedocs.org/projects/tus-py-client/), I've set the metadata to point to this documentation as the homepage but maybe that's a bad idea.